### ISSUE TITLE
Amortissement et durée de simulation

### ISSUE THREAD

**Issue by laem** [betagouv/ecolab-data#816]
Le débat est vif sur le sujet de la fonction d'amortissement de l'empreinte de la construction d'une voiture. Dans la suite on la fixe à 10 tonnes pour simplifier le raisonnement. 

Plusieurs méthodes sont possibles, dont : 
- pas d'amortissement : l'achat déclenche des émissions à court terme, donc aucune raison physique de lisser ces émissions dans le temps
- amortissement linéaire : une voiture tient en moyenne 20 ans, on divise donc 10 par 20. C'est la méthode courante, pratique
- amortissement à taux dégressif : on prend acte que le 1) est correct physiquement mais difficile à gérer socialement (l'année après l'achat, l'empreinte tombe à 0) et que 2) est une astuce comptable pour remettre à demain les émissions d'aujourd'hui

Nous avons choisi 3) aujourd'hui, avec une fonction d'amortissement pas suffisamment justifiée mais publique. 

Je voulais ici souligner un autre point : pour choisir la fonction d'amortissement, il faut prendre en compte le contexte de la simulation. Elle se fait aujourd'hui sur un an, année n. Il apparaît ainsi clairement que la méthode linéaire renvoie l'essentiel des émissions qui pourraient être décidées en année n en dehors de la période de simulation. 

Je fais donc l'hypothèse que l'amortissement linéaire n'est pertinent que dans un simulateur qui met en contexte la simulation dans un temps plus long (décennie par ex.) en complément du temps court de l'année. 

Qu'en pensez-vous ?

**Comment by Petbaxe** [comment]
Je suis plus favorable à la méthode linéaire qui me parait la plus simple et donc la plus compréhensible et donc la plus transparente. Avec la méthode trois on a le sentiment d'un choix arbitraire ...
De plus il est écrit dans la documentation à la section amortissement particulier :
"Quand on achète une voiture, on fait émettre une quantité donnée de GES. Disons par exemple, 10 tonnes, pour une durée de vie de la voiture de 10 ans. Ces gaz sont émis l'année de l'achat, ou plus ou moins (certains composants auront déjà été produits)."
Or cette durée de 10 correspond à l'âge **moyen** et non à la **durée de vie moyenne** !!! qui peut être environ du double.
Voir cette source : https://www.largus.fr/actualite-automobile/le-top-15-des-voitures-envoyees-a-la-casse-10348407.html
Le minimum serait donc de reprendre la règle 3 de l'amortissement à taux dégressif mais de l'appliquer sur 20 ans et non 10.
Cordialement

**Comment by rdub975** [comment]
Perso la méthode 2 est plus vraie car l'achat d'une voiture est quelque chose qui va être répétée plusieurs fois au cours de la vie d'un individu. Ce qui doit changer c'est éventuellement qu'au cours de sa vie il en achète moins (genre passer de changer sa voiture tous les 10 ans au lieu de tous les 5 ans), et d'ailleurs on est déjà dans cette dynamique là avec un allongement de l'age moyen des véhicules sur nos routes (5.9 ans en 1990, 8.9 en 2015 : https://fr.wikipedia.org/wiki/Parc_automobile_fran%C3%A7ais). Surtout l'utilité du message "vous venez d'émettre 10 tonnes" de CO2 est quasi nulle, vu que cette opération ne se reproduira pas avant des années, il a DEJA acheté. La personne qui projette d'acheter un nouveau véhicule mais ne l'a pas encore fait ne peut même pas être dissuadé vu que son chiffre à lui est resté faible avec son ancienne voiture bien amortie. 

Ensuite, même avec l'application de la méthode 3). En prenant votre texte d'explication et en modifiant cette fois l'amortissement sur 20 ans et non 10, on se trouve avec 20% du CO2 la première année (40% sur 2 ans et non plus 1), soit 2.4t en cas de véhicule électrique et 1.4 en cas de véhicule thermique au lieu de 0.5t avec la méthode linéaire. Ca n'a rien a voir avec les 3 à 5t qui sont annoncé actuellement à l'utilisateur et le "message" n'a pas l'impact que vous attendez.  

Je ne vois pas ce qui justifie de ne pas utiliser la méthode linéaire

**Comment by laem** [comment]
> Or cette durée de 10 correspond à l'âge moyen et non à la durée de vie moyenne !!! qui peut être environ du double.

Le sujet de la durée de vie est un bug déjà identifié, qui concerne toutes les méthodes d'amortissement. Ça ne sert à rien de mettre en gras dans cette issue ce qui est écrit dans une autre plus pertinente. Si tu peux modifier ton commentaire pour faire une référence, la discussion sera plus claire 🙂

**Comment by laem** [comment]
Ce problème de l'amortissement d'émissions en année n dont les bénéfices sont toujours tirés en années n+1 ... n+m se retrouve en fait sur un tas de sujets. 

Exemple ici #862 sur le retour d'un utilisateur  : 

> Je ne prends pas l'avion tous les ans, mais j'ai fait un tour du monde en avion il y a 3 ans. dans votre test j'ai mis zéro, ca me donne un très bon résultat, que je sais faux. donc je vous suggérer de le lisser sur une plus longue période, par ex 3 ou 5ans, en nombre d'heures de vol. je crois qu'un nombre important de Francais se paie un voyage lointain non annuel, notamment dans les classes moyennes. 

En prenant l'avion en année n, puis rien les trois années suivantes, puis avion en n+4, on se retrouve mal à l'aise face à la question qui est annuelle. Aujourd'hui, on attribue toutes les émissions d'un vol d'avion à l'année n, puis rien jusqu'à n+4, alors qu'on pourrait considérer que la maille de simulation pertinente c'est les 5 ans ou 10 ans, où ces émissions seraient simplement cumulées et représentatives de la moyenne temporelle. 

Idem pour le smartphone, le canapé, la construction de maison, etc

**Comment by mrubioroy** [comment]
Bonjour,

Je pense que les deux modes de calcul ont leur intérêt. Calculer en moyenne temporelle, lissant sur plusieurs années, est intéressant car ça donne une première idée de notre impact global, indépendamment des aléas de chaque année.
Par contre, le calcul "réel" a aussi son intérêt. Si chaque année je fais le calcul pour suivre mon évolution, l'attribution des émissions réelles (les 10 tonnes si achat de voiture) a son sens. Aussi, cette méthode permet de bien calculer l'empreinte d'un "patrimoine" croissant. Je ne sais pas si permettre les deux modes introduirait trop de complexité d'utilisation.

Sinon, pour un calcul "moyenné", je ne pense pas que l'amortissement linéaire soit approprié. Si nous prenons l'exemple de la voiture, plus l'achat neuf est fréquent, plus on monte la pression sur le parc vers la mise à la casse des voitures plus anciennes, qui pourtant pourraient encore servir. Ce raisonnement est valable pour tout type de bien.

Il faut donc un amortissement dégressif, par exemple en exponentielle décroissante, dont l'intégrale sur la durée de vie correspondrait aux émissions de fabrication :
<img src="https://user-images.githubusercontent.com/7759830/107379537-4e657f00-6aed-11eb-982c-6d72262ae8c5.jpg" height="150">

où _e(t)_ est les émissions pour l'année _t_, _t_D_ est la durée de vie (années), _E_ est les émissions de fabrication, et _k_ est une constante permettant d'ajuster la dégressivité (_k=4_ donne de bons résultats). La somme des contributions de chaque année (de _0_ à _t_D_-_1_) vaut toujours _E_.

**Comment by laem** [comment]
> Je pense que les deux modes de calcul ont leur intérêt. 
Idem, je ne suis pas certain qu'on puisse trancher, ce sont juste deux points de vue complémentaires.
>Je ne sais pas si permettre les deux modes introduirait trop de complexité d'utilisation.
Je pense que nous sommes bien placés ici dans nosgestesclimat.fr pour tenter une expérience d'utilisation qui combine ces deux points :) 

> par exemple en exponentielle décroissante, dont l'intégrale sur la durée de vie correspondrait aux émissions de fabrication :

Il me semble que c'est une version plus rigoureuse et régulière de ce qui a été retenu (dans l'"urgence") pour l'instant.

```yaml

 formule: 
    variations: 
      - si: âge < 1
        alors: 40%
      - si: âge < 2
        alors: 20%
      - si: âge < 3
        alors: 10%
      - si: âge < 10
        alors: 5%
      - sinon: 0%
      
      ```
Merci pour les remarques et cette proposition de fonction !

**Comment by mrubioroy** [comment]
> * amortissement linéaire : une voiture tient en moyenne 20 ans, on divise donc 10 par 20. C'est la méthode courante, pratique

Effectivement, en 2018 l'age moyen de mise à la casse était de 19 ans [[Le Parisien]](https://www.leparisien.fr/economie/prime-a-la-conversion-les-demandes-en-hausse-de-64-27-04-2019-8061672.php). Donc _t_D_ = _20_. Il faudrait faire le même exercice pour plein d'autres biens...

**Comment by laem** [comment]
> Effectivement, en 2018 l'age moyen de mise à la casse était de 19 ans

A ce sujet, je ne pige pas pourquoi les études se basent sur une durée de vie de 10 ans en général. Par exemple l'étude FNH-Carbone4. Peut-être parce que l'essentiel de l'usage se fait dans les premières 10 années, le reste étant anecdotique ? Je n'ai rien pour confirmer ça.

**Comment by mrubioroy** [comment]
> Merci pour les remarques et cette proposition de fonction !

J'essaye de faire un pull request mais je n'arrive pas à écrire la formule dans le bac à sable public.codes : exp(x), x**y, x^y, pow(x,y) rien ne fonctionne ! J'ai l'impression que l'interprète n'attend que * / + -

**Comment by laem** [comment]
> J'essaye de faire un pull request mais je n'arrive pas à écrire la formule dans le bac à sable public.codes : exp(x), x**y, x^y, pow(x,y) rien ne fonctionne ! J'ai l'impression que l'interprète n'attend que * / + -

Ah, désolé, les fonctions déjà disponibles dans publicodes sont assez partielles. Elles viennent du droit des prélèvements sociaux, aucune chance que les législateurs aient implémenté une fonction si évoluée :smile: 

Il va donc falloir, si on retient cette fonction d'amortissement (on en parle cet aprem), qu'on implémente un mécanisme dédié, qui sera codé en JS, documenté et testé.

**Comment by laem** [comment]
Voici quelques une tentative de synthèse de la situation : https://hackmd.io/@laem/amortissement-climat

**Comment by martinregner** [comment]
Eléments complémentaire sur l'amortissement linéaire. Il se trouve que c'est celui choisi dans la méthode Bilan Carbone, et dans la Base Carbone (ex. calcul des FE pour les déplacements en kgCO2/pass.km : la fabrication est amortie linéairement sur la durée de vie du moyen de transport).
C’est donc la méthode qui permettra de garder la meilleure cohérence d'ensemble, tout en restant la plus facile à expliquer.

Il serait néanmoins intéressant de mentionner dans la FAQ qu’il y a des méthodes différentes possibles, et d'expliquer notre choix.

Le point clé reste en effet dans tous les cas la pédagogie à ajouter sur ce sujet.

**Comment by laem** [comment]
> Eléments complémentaire sur l'amortissement linéaire. Il se trouve que c'est celui choisi dans la méthode Bilan Carbone, et dans la Base Carbone (ex. calcul des FE pour les déplacements en kgCO2/pass.km : la fabrication est amortie linéairement sur la durée de vie du moyen de transport).

Pour l'empreinte en question, par exemple de la voiture électrique, le calcul est j'imagine le suivant : 

``` 
Empreinte usage + empreinte fabrication / durée de vie. 

Donc par exemple 0.3 kgCO2e / km + 10 000 kgCO2e / 200 000km.
```

La question me semble légèrement différente dans ce fil. La base bilan-GES n'a je crois pas le loisir de faire autrement. 
L'unité du FE est en /km, donc forcément la fabrication doit être ramenée en km de cette façon. 

Or, nous introduisons justement dans nosgestesclimat le paramètre de l'âge de la voiture, entre autres, et c'est bien cette information en plus qu'il faut gérer, car elle est différentiante : en appliquant le même principe mais sur la durée (20 ans au lieu de 200k km), on fait un choix qu'on aurait pu faire différemment.

**Comment by martinregner** [comment]
Le choix de la "simplicité" du linéaire peut aussi être à rapprocher des incertitudes fortes associées aux FE : il faut garder en tête que l'on travaille vraiment avec des ordres de grandeur, qui restent donc adaptés à des méthodes "simples" d'amortissement.
Il nous est clairement déconseillé de recourir à des méthodes plus sophistiquées qui pourraient à tort donner l'impression que les données sont d'une grande précision.

**Comment by MisterretsiM13** [comment]
Pour apporter de l'eau au moulin, même si je suis en accord avec le choix de la "simplicité" évoqué par @martinregner : la nouvelle réglementation environnementale des bâtiments neufs utilisent l'ACV "dynamique", avec un facteur correctif selon l'année d'émission (voir https://www.cerema.fr/system/files/documents/2022/02/acv_dynamique_re2020.pdf#page=7).

Cependant, cette méthode ne semble pas encore faire consensus parmi les chercheurs : voir cette note qui approfondit les biais de la formule retenue dans la réglementation : https://www.univ-gustave-eiffel.fr/fileadmin/Fichiers/Universite_Gustave_Eiffel/Documents/Note_Univ_Gustave_Eiffel_Analyse_du_Cycle_de_Vie_pour_la_RE2020.pdf

Avec notamment une des conclusions intéressante :
> "Le choix d’une méthode dynamique reste en premier lieu un choix politique, compte tenu de l’effet peu
significatif sur le changement climatique de l’étalement des émissions de quelques dizaines d’années. Ce choix
est destiné à inciter les acteurs économiques à retarder leurs émissions de GES."

**Comment by laem** [comment]
Merci pour la trouvaille de questionnements similaires aux notres :)

**Comment by lbranaa** [comment]
https://github.com/datagir/nosgestesclimat/issues/1673 #1313

**Comment by Marc-marc-marc** [comment]
le problème que je souligne dans #1673 c'est le problème de double comptabilité lié aux objets dont la production date d'avant l'année de l’établissement du budget carbone restant pour une trajectoire de 1.5° 1.75° 2° : 
si le budget carbone en 2020 est de X TeqCo2 "à vie" ce qui signifie Y TeqCo2 par an, en simulant une année postérieure à 202, on ne peux pas y inclure un amortissement d'une production ayant déjà été intégralement comptabilisé dans le budget carbone restant sans causer une double comptabilisation, ceci vaut quasi uniquement pour les objets "éternel" tel qu'un logement.
pour les objets à courte durée de vie (à mes yeux, les objets qui seront produit plus d'une fois d'ici 2030-2050, y compris les voitures ou gros électro), c'est plus compliqué et l'idéal serrait un double affichage : 
votre bilan carbone de cette année est X TeqCo2
votre bilan carbone moyen est de Y TeqCo2
ou un choix : voulez faire le bilan carbone de cette année ou un moyenne lissée ?

**Comment by laem** [comment]
Oui, proposer les deux options serait techniquement correct... mais incompréhensible pour l'utilisateur tel que. 

C'est pour cette raison que nous n'avons pas trop avancé sur les problèmes soulevés dans cette issue : ça demande un travail d'UX et de pédagogie important.
